assets/project.js

```diff
@@ -82,4 +128,5 @@ function initMap() {
                             'Error: The Geolocation service failed.' :
                             'Error: Your browser doesn\'t support geolocation.');
       infoWindow.open(map);
-}
```

Comment:
I'm not sure why it's keeping us from resolving this, but until this deletion is fixed, we can't push this out to the master.